app/index.js

```diff
@@ -13,24 +11,18 @@ if (!config.SLACK_CLIENT_ID || !config.SLACK_SECRET) {
   process.exit(-1);
 }
 
-const PORT = process.env.PORT || 7142;
-
-// Allows dependency injection (injected modules need to be in the same directory as this source)
-const OAUTH_HANDLER_SOURCE = process.env.OAUTH_HANDLER_SOURCE || 'oauth-handler.js';
-const WEBSERVER_EXTENSIONS_SOURCE = process.env.WEBSERVER_EXTENSIONS_SOURCE || 'extensions.js';
-const STORAGE_MIDDLEWARE_SOURCE = process.env.STORAGE_MIDDLEWARE_SOURCE || 'no-storage.js';
+const trustpilotApi = require('./trustpilot-api')(config);
+const storageEngine = require('./storage-engine');
+const slackapp = require('./slackapp.js')(config, trustpilotApi, storageEngine);
+const webserverExtensions = require('./webserver-extensions');
+const oAuthCallback = require('./oauth-callback');
```

Comment:
Having to override imports or rename imported files to get the correct `storageEngine` or `webserverExtensions` seems a little awkward. It might be better to have flags/environment variables which are accessed within this module to instantiate the correct instance of these modules.